wp-content/themes/twentytwenty-child/student-shorcode-page.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/* 
+ * Template Name: Student Shortcode Page
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty
+ * @since Twenty Twenty 1.0
+*/
+get_header(); ?>
+
+<header class="archive-header has-text-align-center header-footer-group">
+    <div class="archive-header-inner section-inner medium">
+        <a class="archive-title" href="<?php home_url('students-shortcode'); ?>" >Students Shortcode</a>
+    </div>
+</header>
+
+<div id="site-content" class="<?php echo is_page_template( 'templates/template-full-width.php' ) ? '' : 'thin'; ?>">
```

Comment:
we do not need this check here, as the page template is different, as you are creating it